### PR TITLE
use uselagoon for lagoon-core

### PR DESCRIPTION
**Pull request by Schnitzel**

### CODE REVIEW

File: charts/lagoon-core/values.yaml [hunk 1]
```diff
@@ -595,7 +595,7 @@ drushAlias:
 logsDBCurator:
   enabled: true
   image:
-    repository: amazeeiolagoon/logs-db-curator
+    repository: uselagoon/logs-db-curator
     pullPolicy: Always
     # Overrides the image tag whose default is the chart appVersion.
     tag: ""
```
Comment (tobybellwood) [inline charts/lagoon-core/values.yaml:601]: Until https://github.com/amazeeio/lagoon/pull/2409 is released - we have to use the testlagoon version of this image
```suggestion
    repository: testlagoon/logs-db-curator
    pullPolicy: Always
    # Overrides the image tag whose default is the chart appVersion.
    tag: "pr-2409"
```

File: charts/lagoon-core/values.yaml [hunk 2]
```diff
@@ -639,7 +639,7 @@ sshPortal:
   enabled: false
   replicaCount: 2
   image:
-    repository: amazeeiolagoon/ssh-portal
+    repository: uselagoon/ssh-portal
```
Comment (tobybellwood) [inline charts/lagoon-core/values.yaml:642]: pr-branches are only in testlagoon
```suggestion
    repository: testlagoon/ssh-portal
```

**Comment by tobybellwood** [comment]
This is the first step towards the larger PR in #183 .

Note that the `main` imageTag doesn't exist in the `uselagoon` repo - so any cluster that builds off main will also need to provide `testlagoon` based repository overrides in their values files (or we push main branch into the `uselagoon` namespace too?)

**Comment by Schnitzel** [comment]
@tobybellwood 
oh wow, sorry didn't see #183 when I pushed that one.

Let's close this one and work on #183

**Pull request closed without merging**